### PR TITLE
jpegturbo Issue 18

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
I guess it isn't so much about being "smart" as it working around issues i've seen before with conda_config_yaml

**Comment by hmaarrfk** [comment]
i think that zipping keys, that are in the middle of migrations is just buggy.

File: .ci_support/osx_64_r_base4.1.yaml [hunk 1]
```diff
@@ -4,12 +4,24 @@ c_compiler:
...
 cran_mirror:
 - https://cran.r-project.org
+freetype:
+- '2'
+harfbuzz:
+- '6'
+icu:
+- '70'
+libjpeg_turbo:
+- 2.1.5
```
Comment (mfansler) [inline .ci_support/osx_64_r_base4.1.yaml:22]: I'm confused why this is here (R4.1) and yet the recipe still works. But I've checked the artifacts and everything looks good so... 🤷

File: conda-forge.yml [hunk 2]
```diff
@@ -1,3 +1,5 @@
+azure:
+  store_build_artifacts: true
```
Comment (mfansler) [inline conda-forge.yml]: Guess we should drop this now that everything appears to be working.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -37,15 +37,22 @@ requirements:
...
-    - xorg-libxt                 # [not win]
-
+    - freetype
+    - harfbuzz
+    - icu
+    {% if r_base == "4.1" %}
+    - jpeg 9
+    {% else %}
+    - libjpeg-turbo
+    {% endif %}
```
Comment (mfansler) [inline recipe/meta.yaml:50]: Nice. Apparently, this also has the advantage of not triggering the linter bug.

**Comment by mfansler** [comment]
@conda-forge-admin please rerender

File: .ci_support/osx_64_r_base4.1.yaml (hunk 1 above)
Comment (hmaarrfk) [reply .ci_support/osx_64_r_base4.1.yaml:22]: yeah, the rendering will try to  specify everything it sees in the recipe. not too smart.

**Comment by hmaarrfk** [comment]
I never asked:

is there some incompatibility between r 4.1 and turbo? turbo is pretty widely used.

**Review by mfansler — APPROVED** [review]

**Comment by mfansler** [comment]
> I never asked:
> 
> is there some incompatibility between r 4.1 and turbo? turbo is pretty widely used.

[The migration](https://github.com/conda-forge/r-base-feedstock/pull/243) is failing for other reasons (Windows no longer builds). The splitting here is just temporary until that moves forward.

**Comment by mfansler** [comment]
@hmaarrfk thanks so much for the help on this! Very enlightening to see some more advanced tricks-of-the-trade in action.

**Pull request merged**